islandora_compound_object.module

```diff
@@ -568,10 +568,17 @@ function islandora_compound_object_menu_local_tasks_alter(&$data, $router_item,
       $compound_object = menu_get_object('islandora_object', 2);
       $compound_info = islandora_compound_object_retrieve_compound_info($compound_object, TRUE);
       if ($compound_info) {
-        drupal_set_title(t('@parent - @child', array(
-          '@parent' => $compound_info['parent_label'],
-          '@child' => $compound_info['label'],
-        )), PASS_THROUGH);
+        if ($compound_object->id == $compound_info['parent_pid']) {
+          // We are the parent so don't repeat the label twice.
+          drupal_set_title($compound_info['parent_label']);
```

Comment:
Should we @ escape the $compound_info['parent_label'] too? Just to make sure it is consistent with the existing option?

Reply:
[drupal_set_title()](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/drupal_set_title/7.x) sanitizes the string by default, in the other one we use [t()](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/t/7.x) to do the concatenation and sanitization, then pass `PASS_THROUGH` to disable sanitization in `drupal_set_title()`. 

In the new one we don't set `PASS_THROUGH` so it will default to `CHECK_PLAIN` which is the same sanitization that `@` in [format_string()](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/format_string/7.x) does.

We could do: 
```php
drupal_set_title(t('@label', array('@label' => $compound_info['label'])), PASS_THROUGH);
```

but it seems like a lot of work for a string that can't be translated, since its a variable.

Reply:
@jonathangreen totally. I think I even made a pull for the same stuff a year or more ago! gosh, getting old. Cool. Will merge in 24 hours. thanks a lot